### DISCUSSION TITLE
JSVM namspace net documentation

### DISCUSSION THREAD

**Discussion by helander** [pocketbase/pocketbase#3317] [Q&A] [answered]
The documentation for namespace net in JSVM only includes Golang code examples. Are there any examples in Javascript?
I am interested in setting up a UDP listener (preferably that it returns both unicast and broadcast traffic).

**Comment by ganigeorgiev** [comment, marked as answer]
This is not possible at the moment with only the JavaScript hooks and you'll have to use Go if you need more advanced control over the application flow.

Note that [JSVM reference docs](https://pocketbase.io/jsvm/index.html) are autogenerated from the Go code (_via [pocketbase/tygoja](https://github.com/pocketbase/tygoja)_) and not all listed types (_those outside of the "PocketBase" section_) are directly accessible or callable (_the generated types are usually based on Go struct field types, function/method arguments and return types_).
